src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -667,6 +667,7 @@ class DocBaseViewer extends BaseViewer {
                 if (this.shouldThumbnailsBeToggled()) {
                     this.rootEl.classList.add(CLASS_BOX_PREVIEW_THUMBNAILS_OPEN);
                     this.emit(VIEWER_EVENT.thumbnailsOpen);
+                    this.resize();
```

Comment:
Now that we aren't transitioning on initial load, a resize without a timeout will sufficiently resize the preload or the doc depending on what's ready.